stl/inc/optional

```diff
@@ -617,6 +641,22 @@ _NODISCARD constexpr bool operator>=(const _Ty1& _Left, const optional<_Ty2>& _R
     return _Right ? _Left >= *_Right : true;
 }
 
+#ifdef __cpp_lib_concepts
+// clang-format off
+template <class _Ty1, class _Ty2>
+    requires (!_Is_specialization_v<_Ty2, optional>) // TRANSITION, LWG issue not yet submitted

```

Comment:
Do we have a GH issue to reduce this constraint recursion and file an LWG issue? I just invested about 30 minutes before deciding I don't have time to file it now, but want to be sure it isn't dropped.

Reply:
On second thought, it's very unlikely that someone has filed this GH issue, I'll do so now.

Reply:
https://github.com/microsoft/STL/issues/1674